scripts/install2

```diff
@@ -79,10 +79,20 @@ fi
 
 # Get tools
 if ! [ "$(which nodejs)" ]; then
-    curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
-    sudo apt-get install nodejs -y
-fi
 
+#check for armv6 and install it manually instead
```

Comment:
Consistent with other comments please:

`# Check ...`

And replace "it" with "nodejs"?